install.sh

```diff
@@ -164,6 +166,35 @@ if [ "$SENTRY_DATA_NEEDS_MIGRATION" ]; then
     "mkdir -p /tmp/files; mv /data/* /tmp/files/; mv /tmp/files /data/files; chown -R sentry:sentry /data"
 fi
 
+
+if [ ! -f "$RELAY_CREDENTIALS_JSON" ]; then
+    echo ""
+    echo "Generating Relay credentials..."
+
+    $dcr --user $(id -u) relay --config /etc/relay credentials generate --overwrite
+    chmod a+r $RELAY_CREDENTIALS_JSON
+    # display the contents of the relay directory (for debug purposes)
+    ls -al ./relay
+    CREDENTIALS=$(sed -n 's/^.*"public_key":[[:space:]]"\([a-zA-Z0-9_-]*\)".*$/\1/p' "$RELAY_CREDENTIALS_JSON")
+    CREDENTIALS="SENTRY_RELAY_WHITELIST_PK = [\"$CREDENTIALS\"]"
+
+    if grep -xq SENTRY_RELAY_WHITELIST_PK "$SENTRY_CONFIG_PY"; then
+        echo "FAIL: SENTRY_RELAY_WHITELIST_PK already exists in $SENTRY_CONFIG_PY, please replace with:"
+        echo ""
+        echo "  $CREDENTIALS"
+        echo ""
+        exit 1
+    fi
+
+     echo "" >> "$SENTRY_CONFIG_PY"
+     echo "$CREDENTIALS" >> "$SENTRY_CONFIG_PY"
+
+     tail -n 5 "$SENTRY_CONFIG_PY"
+else
+     echo ""
+     echo "Relay credentials exist will NOT generate new ones."
+fi
```

Comment:
We don't say anything when we don't generate a secret key (because it already exists). Let's be consistent and say nothing if we do nothing.
```suggestion
fi
```

Reply:
Sorry, I really don't get your point about not sending a message when not generating a secret key. We generate the whole configuration as a unit that includes ( id, public key and secret key). 
To me the message is very informative and make a lot of sense. 

Having a credentials.json already existing at install time would be unusual. 
Maybe the user has already run install.sh previously and is running it again in the hope that everything will be reset. In this case we should mention that we have found some existing keys and we are not going to mess with them again ( so if he has been messing with the Relay keys, re running the script will not help him).

Reply:
We do not output anything when we don't generate a secure key: https://github.com/getsentry/onpremise/blob/c20956527c5560625bb8b83fd605ee0b742a4ccf/install.sh#L93-L102

I'm saying we should be consistent with that behavior and simply don't output anything if we don't do anything. Remember, this maybe useful for _you_ as a developer but this repo is for end users who don't necessarily know or care about Relay or its credentials.